include/fiction/algorithms/simulation/sidb/convert_potential_to_distance.hpp

```diff
@@ -0,0 +1,63 @@
+//
+// Created by Jan Drewniok on 10.11.23.
+//
+
+#ifndef FICTION_CONVERT_POTENTIAL_TO_DISTANCE_HPP
+#define FICTION_CONVERT_POTENTIAL_TO_DISTANCE_HPP
+
+#include "fiction/algorithms/simulation/sidb/sidb_simulation_parameters.hpp"
+#include "fiction/technology/physical_constants.hpp"
+
+#include <cmath>
+#include <cstdint>
+#include <limits>
+#include <vector>
+
+namespace fiction
+{
+/**
+ * The electrostatic potential on hydrogen-passivated Silicon is typically modeled using a screened Coulomb potential.
+ * This electrostatic potential is commonly employed to determine the electrostatic potential for a given distance
+ * (between SiDB and point under consideration) and given physical parameters. However, the function provided here
+ * serves the inverse purpose by calculating the distance for a given potential and given physical parameters.
+ *
+ * @param params The physical parameters for a given hydrogen-passivated Silicon surface.
+ * @param potential The electrostatic potential (unit: V) to be converted to a distance.
+ * @param precision The precision level for the conversion, specifying the number of decimal places.
+ * @return The distance (unit: nm) corresponding to the given electrostatic potential.
+ *
+ * @note The runtime depends on the set precision.
```

Comment:
```suggestion
 * @note The runtime depends on the provided precision.
```

In which way? Linearly? Quadratically? Exponentially?

Also, please move the note up right in between the explanation and the parameters.